src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.model;
+
+import java.util.StringJoiner;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
+    }
+
+    public CinemaHall(int capacity, String description) {
+        this.capacity = capacity;
+        this.description = description;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public int getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity(int capacity) {
+        this.capacity = capacity;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String toString() {
+        return new StringJoiner(", ", CinemaHall.class.getSimpleName() + "[", "]")
+                .add("id=" + id)
+                .add("capacity=" + capacity)
+                .add("description='" + description + "'")
+                .toString();
```

Comment:
let's use simple string concatenation
it will use StringBuilder under the hood in this case